docs/utils.md

```diff
@@ -135,15 +135,27 @@ Common arguments:
 ```
 ```
 freqtrade list-hyperopts --help
-usage: freqtrade list-hyperopts [-h] [-v] [--logfile FILE] [-V] [-c PATH]
-                                [-d PATH] [--userdir PATH]
-                                [--hyperopt-path PATH] [-1]
+usage: freqtrade hyperopt-list [-h] [-v] [--logfile FILE] [-V] [-c PATH]
+                               [-d PATH] [--userdir PATH] [--best]
+                               [--profitable] [--min-avg-time INT]
+                               [--max-avg-time INT] [--min-avg-profit FLOAT]
+                               [--min-total-profit FLOAT] [--no-color]
+                               [--print-json] [--no-details]
```

Comment:
No, `list-hyperopts` is another subcommand (yeah, just found this by myself, they were added separately at different periods of time, with diff. things in the mind...)
See https://www.freqtrade.io/en/latest/utils/#list-hyperopt-results

Reply:
corrected. Updated at wrong place